ERCS/erc-7579.md

```diff
@@ -0,0 +1,351 @@
+---
+eip: 7579
+title: Minimal Modular Smart Accounts
+description: Minimally required interfaces and behavior for modular smart accounts and modules
+author: zeroknots (@zeroknots), Konrad Kopp (@kopy-kat), Taek Lee (@leekt), Fil Makarov (@filmakarov), Elim Poon (@yaonam), Lyu Min (@rockmin216)
+discussions-to: https://ethereum-magicians.org/t/erc-7579-minimal-modular-smart-accounts/17336
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-12-14
+requires: 165, 1271, 2771, 4337
+---
+
+## Abstract
+
+This proposal outlines the minimally required interfaces and behavior for modular smart accounts and modules to ensure interoperability across implementations.
```

Comment:
I'd like to see a bit more technical detail here. The reader should get a high-level overview of how your proposal works just from reading the abstract.